tests/tests_app_examples/test_multi_node.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from tests_app import _PROJECT_ROOT
 
-from lightning_app.testing.testing import application_testing, LightningTestApp
+from lightning.app.testing.testing import application_testing, LightningTestApp
```

Comment:
All examples use `lightning_app`, as imports are modified while packaging.
We need to be consistent throughout.

Since examples are likely to be copy-pasted, and imports at that point would be ran against the `lightning` package (and not `lightning_app`) then I think we should change this throughout.

I suggest to change all imports in a separate PR, and keep this one consistent (`lightning_app`).

Reply:
@lantiga externally, we use lightning.app not lightning_app.

We need a consistent experience for users. Having lightning_app and lightning.app all over the place is already confusing people.

Reply:
@lantiga This is changed only during debugging.